### DISCUSSION TITLE
Playback Speed

### DISCUSSION THREAD

**Discussion by lukmuk** [Nianna/Karedi#79] [Ideas]
Hello Nianna!

I want to try Karedi for creating songs. Is it possible to add adjustable (slower) playback speed to Karedi? 
That would help me in tapping and sometimes note tuning (so far I used UltraStar-Creator for tapping and Yass for editing and both have this feature - so I guess I got used to it).

Thank you for your work and for making Karedi. Runs and feels good so far on Linux!

Kind regards,
lukmuk

**Comment by Nianna** [comment]
Hi lukmuk!

Yes, I think it should be doable, will look into it.

**Comment by Nianna** [comment]
Done - I've added an option to adjust playback speed (to both slower and faster) while maintaining the original pitch.

Feature will be included in next version which I plan to release sometime in November.

**Reply by lukmuk** [reply]
Sounds great, looking forward to the next version!
Thanks for implementing this feature so quickly.
